parsl/dataflow/dflow.py

```diff
@@ -50,7 +51,8 @@ class DataFlowKernel(object):
     """
 
     def __init__(self, config=None, executors=None, lazyErrors=True, appCache=True,
-                 rundir=None, retries=0, checkpointFiles=None, checkpointMode=None):
+                 rundir=None, retries=0, checkpointFiles=None, checkpointMode=None,
+                 db_logger=get_db_logger(level=logging.CRITICAL)):
```

Comment:
Consider allowing the user to disable DB logging by setting this to `None`. Currently it will only be disabled if they don't have `cmreslogging` installed. I think the default should be to have DB logging disabled (the average user is not going to have an AWS instance set up).

Reply:
This is hacky at best. I've set the db_logger to only require the `cmreslogging` module and send out log messages only if logging level is INFO or lower. I think this could use some redesign.